src/main/scala/org/broadinstitute/dsde/rawls/webservice/RawlsApiService.scala

```diff
@@ -458,11 +458,11 @@ trait WorkspaceApiService extends HttpService with PerRequestCreator {
     new ApiResponse(code = 500, message = "Rawls Internal Error")
   ))
   def copyMethodConfigurationRoute = cookie("iPlanetDirectoryPro") { securityTokenCookie =>
-    path("workspaces" / Segment / Segment / "methodconfigs" / "copy" ) { (workspaceNamespace, workspaceName) =>
+    path("workspaces" / Segment / Segment / "methodconfigs" / Segment / Segment / "copy" ) { (workspaceNamespace, workspaceName, methodConfigurationNamespace, methodConfigurationName) =>
       post {
         entity(as[MethodConfigurationName]) { srcMethodConfigurationName =>
           requestContext => perRequest(requestContext, WorkspaceService.props(workspaceServiceConstructor),
-            WorkspaceService.CopyMethodConfiguration(workspaceNamespace, workspaceName, srcMethodConfigurationName))
+            WorkspaceService.CopyMethodConfiguration(MethodConfigurationName(methodConfigurationNamespace,methodConfigurationName,WorkspaceName(workspaceNamespace, workspaceName)), srcMethodConfigurationName))
```

Comment:
I was wondering how this would integrate/conflict with copying from Methods Repo and I think specifying the source in the JSON payload is a good solution.

My preference for endpoint is Doug's because specifying either the source or dest workspace is problematic.  Source because our source might not be a workspace, and dest because `/destWorkspace/methods/copy` reads to me like "copy a method from this workspace" at first glance.